### PR TITLE
Adds Vox Jumpsuits to Loadout Screen - Rebuilt PR #22537 to fix DMI Conflict

### PR DESCRIPTION
**Pull request by PollardTheDragon**
## What Does This PR Do

Adds two otherwise-unused vox uniforms to the species-specific loadout tab.

Enables sensors for said vox uniforms.

Renames "Alien Clothes" to "Vox Jumpsuit" and "Alien Robes" to "Vox Robes"

## Why It's Good For The Game

Takes otherwise unused assets and adds them in a way that is fitting and does not impede gameplay. Adds additional drip, and diversifies the species loadout screen a bit. Modernizes old assets to better fit the current environment.

## Images of changes

image
image
image
image

## Testing

Started many rounds, selected loadout gear, spawned with gear, set sensors to max.

## Changelog
:cl:
tweak: Renamed alien clothes to more fitting Vox Jumpsuit
tweak: Renamed alien robes to more fitting Vox Robes
tweak: Vox robes are now a suit slot item
tweak: Updated sprites
add: Vox robes now have a toggleable hood
add: Added vox clothes to loadout screen
tweak: Gave vox clothes suit sensors 
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
PR related to https://github.com/ParadiseSS13/Paradise/pull/22537

File: code/modules/clothing/spacesuits/alien_suits.dm [hunk 1]
```diff
@@ -143,30 +143,48 @@
...
+	hoodtype = /obj/item/clothing/head/hooded/vox_robe_hood
+	species_restricted = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
+	)
+
+//Vox Robes Hood
+/obj/item/clothing/head/hooded/vox_robe_hood
+	name = "Vox hood"
+	desc = "It's hood that covers the head, made of a weird and flowing material."
```
Comment (Henri215) [inline code/modules/clothing/spacesuits/alien_suits.dm:177]: See if someone from the lore team has any description suggestions, its a good place for pieces of the species lore.

Also, which hood doesnt cover the head??

File: code/modules/clothing/spacesuits/alien_suits.dm [hunk 2]
```diff
@@ -143,30 +143,48 @@
...
-	name = "alien clothing"
+	name = "Vox jumpsuit"
 	desc = "This doesn't look very comfortable."
-	icon_state = "vox-casual-1"
-	item_color = "vox-casual-1"
-	item_state = "vox-casual-1"
+	icon_state = "vox-jumpsuit_s"
+	item_color = "vox-jumpsuit"
+	item_state = "vox-jumpsuit"
 	body_parts_covered = LEGS
```
Comment (Contrabang) [inline code/modules/clothing/spacesuits/alien_suits.dm:158]: This should probably be removed, it looks like it covered all the default jumpsuit spots, which is `body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS`
```suggestion
```

Comment (PollardTheDragon) [reply code/modules/clothing/spacesuits/alien_suits.dm:158]: Fair enough. It's an artifact from the old version, and it makes sense with the sprite update.

File: code/modules/clothing/spacesuits/alien_suits.dm (hunk 1 above)
Comment (PollardTheDragon) [reply code/modules/clothing/spacesuits/alien_suits.dm:176]: Loreful descriptions have been added, with thanks to Octus for assisting with lore.

File: code/modules/clothing/spacesuits/alien_suits.dm [hunk 3]
```diff
@@ -143,30 +143,47 @@
...
-	item_state = "vox-casual-2"
+	name = "Vox jumpsuit"
+	desc = "These loose clothes are optimized for the labors of the lower castes onboard the arkships. Large openings in the top allow for breathability while the pants are durable yet flexible enough to not restrict movement."
+	icon_state = "vox-jumpsuit_s"
+	item_color = "vox-jumpsuit"
+	item_state = "vox-jumpsuit"
+
+/obj/item/clothing/suit/hooded/vox_robes
+	name = "Vox robes"
+	desc = "Large, comfortable robes worn by those who need a bit more covering. The thick fabric contains a pocket suitable for those that need their hands free during their work, while the cloth serves to cover scars or other damage to the wearer's body."
```
Comment (Henri215) [inline code/modules/clothing/spacesuits/alien_suits.dm:161]:
```suggestion
	desc = "Large, comfortable robes worn by those who need a bit more covering. The thick fabric contains a pocket suitable for those that need their hands free during their work, while the cloth serves to cover scars or other injuries on the wearer's body."
```

File: code/modules/clothing/spacesuits/alien_suits.dm [hunk 4]
```diff
@@ -143,30 +143,47 @@
...
+	hoodtype = /obj/item/clothing/head/hooded/vox_robe_hood
+	species_restricted = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
+	)
+
+//Vox Robes Hood
+/obj/item/clothing/head/hooded/vox_robe_hood
+	name = "Vox hood"
+	desc = "The thick fabric of this hood serves a variety of purposes to the vox wearing it - serving as a method to hide a scarred face or way to keep warm in the colder areas onboard the ship."
```
Comment (Henri215) [inline code/modules/clothing/spacesuits/alien_suits.dm:176]:
```suggestion
	desc = "The thick fabric of this hood serves a variety of purposes to the vox wearing it - serving as a method to hide a scarred face or a way to keep warm in the coldest areas onboard the ship."
```

**Review by Henri215 — COMMENTED** [review]
Also.. the correct word to be used here is onboard, on board, on-board or aboard? I dont know :D

**Comment by PollardTheDragon** [comment]
Behold, my lack of grammar skills, fixed.

File: code/modules/clothing/spacesuits/alien_suits.dm [hunk 5]
```diff
@@ -143,30 +143,47 @@
...
+	hoodtype = /obj/item/clothing/head/hooded/vox_robe_hood
+	species_restricted = list("Vox")
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
+	)
+
+//Vox Robes Hood
+/obj/item/clothing/head/hooded/vox_robe_hood
+	name = "Vox hood"
+	desc = "The thick fabric of this hood serves a variety of purposes to the vox wearing it - serving as a method to hide a scarred face or a way to keep warm in the coldest areas onboard the ship."
```
Comment (Henri215) [inline code/modules/clothing/spacesuits/alien_suits.dm:176]: The correct term is "onboard" here?

Comment (PollardTheDragon) [reply code/modules/clothing/spacesuits/alien_suits.dm:176]: Yes. It was a bit confusing, but aboard is referring to the act of actively boarding something, but onboard is when something is already on or in a vehicle. "Come aboard" versus "Onboard motor". I have adjusted descriptions to remedy the inconsistencies to be "onboard".

**Review by farie82 — APPROVED** [review]

**Comment by PhantornRU** [comment]
The Vox Jumpsuits have bad translucent pixels.

**Pull request merged**